### ISSUE TITLE
What is the license for this code?

### ISSUE THREAD

**Issue by robinglord1** [bthyreau/hippodeep_pytorch#4]
Hi,
First of all, thanks a lot for this code and model.
I was wondering if this `hippodeep_pytorch` Github project is under the same MIT license as was the inital `hippodeep` code/model (https://github.com/bthyreau/hippodeep/blob/master/LICENSE) ?
Thank you for the information

**Comment by bthyreau** [comment]
You're right, the license is the same. Thanks for your comment and reminder.
